doc/guides/networking/net-stack-architecture.rst

```diff
@@ -0,0 +1,137 @@
+.. _network_stack_architecture:
+
+Network Stack Architecture
+##########################
+
+High level overview of the network stack
+****************************************
+
+.. figure:: zephyr_netstack_overview.svg
+    :align: left
```

Comment:
output looks weird, probably because of this align you have.

Reply:
Yes, I know. The idea was to place the image to one side and the text to other so that it is easier to understand the architecture in one glance. Unfortunately it seems to require some voodoo css coding that I do not know, so lets center the images then.